README.md

```diff
@@ -83,20 +105,57 @@ var app = express();
 ```
 
 ```js
-// Your answer...
+
+//read
+app.get("/authors", function(res, req){
+  Author.find().then(function(authors){
+    res.render("authors-index", {
+      authors: authors
+    });
+  });
+});
+
+//create
+app.get("/authors/:name", function(res, req){
+  Author.findOne({name: req.params.name}).then(function(author){
+    res.render("authors-show", {
+      author: author
+    });
+  });
+});
+
+//update
+app.put("/authors/:name", function(res, req){
+  Author.findOneAndUpdate({name: req.params.name}, req.body.author, {new: true}).then(function(author){
+    res.render("authors-show", {
+      author: author
+    });
+  });
+});
+
+//delete
+app.delete("/authors/:name", function(res, req){
+  Author.findOneAndRemove({name: req.params.name}).then(function(){
+    res.redirect("/authors")
+  });
+});
 ```
 ### Question #7
 
 Describe the differences between Express and Rails as backend frameworks.
 
 ```text
 
+Express is a lot faster since it requires more legwork (setting up connections, requiring dependencies, files, etc.); whereas, Rails is nicer, it does a lot of the work for you (ALL the files, automatic connections(as long as convention is followed), etc.)
+
 ```
 
 ### Question #8
 
-What is the importance of using body-parser in our express application for post requests? 
+What is the importance of using body-parser in our express application for post requests?
 
 ```js
 
+Body parser is used read the submitted contents of an HTML form.
+
```

Comment:
Not just HTML forms. It's necessary for AJAX as well -- or any situation in which data is being POSTed.